### PR TITLE
adding csd upgrade instructions for 3.0

### PR DESCRIPTION
**Pull request by dereklwood**
- [x] updated integrations guide to reflect Web-App -> UI change
   - [ ] except for the Verification section, will work on that next
- [x] added section on upgrading via Cloudera Manager

### CODE REVIEW

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 1]
```diff
@@ -41,7 +41,7 @@ the 'cdap' user installed by the parcel.
 Prerequisites
 =======================================
 
-#. Node.js (version 0.8.16 through 0.10.37) must be installed on the node(s) where the Web-App
+#. Node.js (version 0.8.16 through 0.10.37) must be installed on the node(s) where the UI
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: Check this with the updated supported versions for the UI, versus the old webapp. Pretty sure it supports other versions of node with the new UI.

File: cdap-docs/integrations/source/partners/cloudera/step-by-step-cloudera.rst [hunk 2]
```diff
@@ -101,7 +101,7 @@ show in the list for the cluster where you installed it. You can select it, and
 Summary* should all turn green, showing completion of startup. 
 
 The *Quick Links* includes a link to the **CDAP Console**, which by default is running on
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/step-by-step-cloudera.rst]: **CDAP UI**

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 3]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
 
 #. You have now completed verification of the installation.
 
+Upgrading an Existing Version
+=======================================
+
+.. rubric:: Upgrading Patch Release versions
+
+When a new compatible CDAP parcel is released, it will be available via the Parcels page in the Cloudera Manager UI
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst (hunk 1 above)
Comment (dereklwood) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: it's in many other places in the doc, so will not hold up this PR for it.  I'm asking the UI team.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 4]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+
+#. Stop all Flows, Services, and other Programs in all your applications.
+
+#. Stop CDAP services
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 5]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+
+#. Use the Cloudera Manager UI to download, distribute, and activate the parcel on all cluster hosts
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 6]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+
+#. Start CDAP services
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 7]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+
+.. rubric:: Upgrading Major/Minor Release versions
+
+These steps will upgrade from CDAP 2.8.0 to CDAP 3.0.0. (**Note:** Apps need to be both recompiled and re-deployed.)
+
+#. Stop all Flows, Services, and other Programs in all your applications.
+
+#. Stop CDAP services
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 8]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+
+#. Ensure your installed version of the CSD matches the target version of CDAP. For example, CSD version 3.0.* is compatible
+   with CDAP version 3.0.*.  Download the latest version of the CSD `here <http://cask.co/resources/#cdap-integrations>`__.
+
+#. Use the Cloudera Manager UI to download, distribute, and activate the parcel on all cluster hosts
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 9]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+
+#. Before starting services, run the Upgrade Tool to update any necessary CDAP table definitions.  From the CDAP Service page,
+   select "Run CDAP Upgrade Tool" from the Actions menu
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst [hunk 10]
```diff
@@ -167,6 +167,49 @@ We provide in our SDK pre-built ``.JAR`` files for convenience.
...
+#. Start the CDAP services.  At this point it may be necessary to correct for any changes in the CSD.  For example, if new CDAP services
+   were added or removed, you must add or remove role instances as necessary.  When upgrading from 2.8.0 to 3.0.0, the CDAP Web-App role has
+   been replaced by the CDAP-UI role:
+
+   - From the CDAP Instances page, select Add Role Instances, and choose a host for the CDAP-UI role.
+
+   - From the CDAP Instances page, check the CDAP-Web-App role, and select Delete from the Actions menu.
+
+#. After CDAP services have started, run the Post-Upgrade tool to perform any necessary upgrade steps against the running services.  From the
+   CDAP Service page, select "Run CDAP Post-Upgrade Tasks"
```
Comment (awholegunch) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: add trailing period.

**Comment by awholegunch** [comment]
Minor typos. But the issue of the node version: once you figure out the answer to that, let me know, as there a number of other places in the docs that will require adjusting to keep in sync.

File: cdap-docs/integrations/source/partners/cloudera/configuring.rst (hunk 1 above)
Comment (dereklwood) [inline cdap-docs/integrations/source/partners/cloudera/configuring.rst]: 0.10.* - 0.12.*

@awholegunch will update separately

**Pull request merged**